springboard/tests/test_filters.py

```diff
@@ -77,3 +77,5 @@ def test_unknown_language_fallback(self):
             display_language_name_filter({}, 'kur_IQ'), 'Kurdish')
         self.assertEqual(
             display_language_name_filter({}, 'spa_ES'), u'espa\xf1ol')
+        self.assertEqual(
+            display_language_name_filter({}, 'ckb_IQ'), u'کوردیی ناوەندی')
```

Comment:
This is not valid - use `blah` when running locally and use the value output from the fail to comapare with